modules/storage-persistent-storage-pv.adoc

```diff
@@ -238,7 +238,7 @@ cluster.
 
 |===
 
-You can view the name of the PVC bound to the PV by running:
+You can view the name of the PVC when it is bound to the PV by running the following command:
```

Comment:
```suggestion
You can view the name of the PVC that is bound to the PV by running the following command:
```

You can get the PVC name other ways. This was about getting the PVC associated with a PV.

Reply:
@ShaunaDiaz I'm OK with this change,  just suggested to update the command from "oc get pv <pv-claim>" to "oc get pv <pv-name>"